test/test_mathics_precedence.py

```diff
@@ -97,7 +97,7 @@
     # RoundImplies[a,UpTee[b,c]],
     # But in Mathics as
     # UpTee[RoundImplies[a, b], c]
-    #    "RoundImplies", # WMA->240, Mathics->200, Must be ~193
+    "RoundImplies",  # WMA->240, Mathics->200, Must be ~193
```

Comment:
Why must it be ~193?  Please write an isolated test that demonstrates this. Thanks.